beacon-chain/state/stategen/epoch_boundary_state_cache.go

```diff
@@ -152,11 +152,11 @@ func (e *epochBoundaryState) put(r [32]byte, s state.BeaconState) error {
 }
 
 // delete the state from the epoch boundary state cache.
-func (e *epochBoundaryState) delete(r [32]byte) error {
+func (e *epochBoundaryState) delete(blockRoot [32]byte) error {
```

Comment:
Same here, this was a good renaming as it added context that was not provided by the method name or godoc comment.